doc/adrs/replace-migration-scaffold-with-clean-scaffold-in-master-branch.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "ADR: Replace migration Scaffold with clean Scaffold in Master Branch"
+date: 2018-08-16
+draft: false
+status: "Accepted"
+---
+
+# ADR: Replace migration Scaffold with clean Scaffold in Master Branch
+
+## Context
+At the moment, the master branch has the scaffold specifically build for helping developers to migrate v1 apps to v2. From monitor, the majority usage comes from [App Migrator](https://github.com/zendesk/zendesk_app_migrator) and [ZAT](https://github.com/zendesk/zendesk_apps_tools) which use the master branch as the download source.
```

Comment:
I believe it should be _built for + NAME_, like built for speed, or in this case _built to help_. Then ditch **to** migrate.
> the master branch has the scaffold specifically built to help developers migrate v1 apps to v2.

Reply:
👍